numpy/__init__.pyi

```diff
@@ -1,8 +1,186 @@
 # very simple, just enough to start running tests
+#
+import builtins
+from typing import Any, Mapping, List, Optional, Tuple, Union
 
-class ndarray: pass
+from numpy.core._internal import _ctypes
+
+class dtype:
+    names: Optional[Tuple[str, ...]]
+    updateifcopy: bool
+    writeable: bool
+    writebackifcopy: bool
```

Comment:
I'm not quite sure where these attributes came from -- I can't find any of `updateifcopy`, `writeable` and `writebackifcopy` on dtype.

Reply:
Actually, these are missing down below on flagsobj.

Reply:
Oops :blush: